engine/common/net_encode.c

```diff
@@ -488,20 +488,20 @@ void Delta_WriteTableField( sizebuf_t *msg, int tableIndex, const delta_t *pFiel
 	Assert( nameIndex >= 0 && nameIndex < dt->maxFields );
 
 	MSG_BeginServerCmd( msg, svc_deltatable );
-	MSG_WriteUBitLong( msg, tableIndex, 4 );	// assume we support 16 network tables
-	MSG_WriteUBitLong( msg, nameIndex, 8 );		// 255 fields by struct should be enough
-	MSG_WriteUBitLong( msg, pField->flags, 10 );	// flags are indicated various input types
-	MSG_WriteUBitLong( msg, pField->bits - 1, 5 );	// max received value is 32 (32 bit)
+	MSG_WriteUBitLong( msg, tableIndex, 4 ); // assume we support 16 network tables
+	MSG_WriteUBitLong( msg, nameIndex, 8 ); // 255 fields by struct should be enough
+	MSG_WriteUBitLong( msg, pField->flags, 10 ); // flags are indicated various input types
+	MSG_WriteUBitLong( msg, pField->bits - 1, 5 ); // max received value is 32 (32 bit)
 
 	// multipliers is null-compressed
-	if( pField->multiplier != 1.0f )
+	if( pField->multiplier <= 0.9999f || pField->multiplier >= 1.0001f )
```

Comment:
Maybe add some macro for such comparsion?
```
#define COMPARE_NEAR(x,val,eps) ((x >((val) - (eps)) ) && (x < ((val) + (eps)) ))
```
if something like this not defined already

Reply:
Yes, I'll add it to mathlib